datastreams/aggregator.go

```diff
@@ -239,6 +240,9 @@ func (a *aggregator) run(tick <-chan time.Time) {
 			a.addKafkaOffset(o)
 		case now := <-tick:
 			a.sendToAgent(a.flush(now))
+		case done := <-a.flushRequest:
+			a.sendToAgent(a.flush(time.Now().Add(bucketDuration * 10)))
+			done <- struct{}{}
```

Comment:
nit: can you close `done` here instead of sending a payload?

Reply:
Updated, thank you!